### PR TITLE
Adds aria-hidden to media player placeholder guidance

### LINKED ISSUE #2580: Update screen reader UX for guidance message in media placeholder
**Is your feature request related to a problem? Please describe.**
We need to update the screen reader UX so that the guidance message is announced with the play button of the media player, as screen reader users can navigate via buttons and may miss this important message otherwise.

**Describe the solution you'd like**
According to the screen reader UX, the guidance message needs to be announced with the play button and before the 'Play video, ...' text, and should follow this format:
```
Guidance, contains strong language and adult humour. Play video, “Video Title”, 1 minute 40 seconds
```
It maybe possible to link up the guidance text that is visually displayed with aria to the play button (instead of passing the guidance text as visually hidden text to the button), though that is only a viable solution if the aria attribute is supported by all supported AT **and** the guidance message is read out first before 'Play video, ...'.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [X] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #2580 and probably #2573 
part of https://github.com/bbc/psammead/pull/2610

**Overall change:** Adds `aria-hidden=true` to media player placeholder guidance

**Code changes:**

- Adds `aria-hidden=true` to media player placeholder guidance

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

**Comment by rhenshaw56** [comment]
This can  be reviewed as it is but should be merged after https://github.com/bbc/psammead/pull/2627

**Review by RayNjeri — APPROVED** [review]

File: packages/components/psammead-media-player/src/Guidance/index.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const GuidanceMessage = styled.strong`
 
 const Guidance = ({ guidanceMessage, service }) => (
   <GuidanceWrapper service={service}>
-    <GuidanceMessage>{guidanceMessage}</GuidanceMessage>
+    <GuidanceMessage aria-hidden="true">{guidanceMessage}</GuidanceMessage>
```
Comment (ghost) [inline packages/components/psammead-media-player/src/Guidance/index.jsx:42]: This is a boolean prop but we are passing a `string`. It should either be `<GuidanceMessage aria-hidden>` or `<GuidanceMessage aria-hidden={true}>`
```suggestion
    <GuidanceMessage aria-hidden>{guidanceMessage}</GuidanceMessage>
```

**Review by simonsinclair — APPROVED** [review]

File: packages/components/psammead-media-player/src/Guidance/index.jsx (hunk 1 above)
Comment (simonsinclair) [reply packages/components/psammead-media-player/src/Guidance/index.jsx:42]: Hi @sadickisaac, aria-hidden can be `true`, `false` or `undefined`.
- `<GuidanceMessage aria-hidden>` is the same as `<GuidanceMessage aria-hidden="true">`
- `<GuidanceMessage aria-hidden="false">` is not the same as `<GuidanceMessage>`

In this case, your suggestion would save us a few bytes, but considering `aria-hidden` can also `="false"` I like the extra verbosity. 😄

**Review by j-pendlebury — APPROVED** [review]
LGTM

**Comment by jamesbrumpton** [comment]
lgtm

**Comment by rhenshaw56** [comment]
will be merged after https://github.com/bbc/psammead/pull/2627

**Pull request merged**